### PR TITLE
HOMS-244 Move assets generation to Dockerfiles

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

**Comment by mxsmirnoff** [comment]
Resolved CHANGELOG conflict

File: Dockerfile.oracle [hunk 1]
```diff
@@ -76,6 +76,9 @@ RUN find config -name '*.sample' | xargs -I{} sh -c 'cp $1 ${1%.*}' -- {}
 RUN mkdir /tmp/config
 RUN cp -r /opt/homs/config/* /tmp/config
 
+RUN yarn install && bundle exec rails assets:precompile
+RUN rm -rf /opt/homs/node_modules/
```
Comment (TimAle) [inline Dockerfile.oracle]: Do it in one `RUN` command, otherwise the image size will not be shrinked after `node_modules` removing.

**Comment by mxsmirnoff** [comment]
Updated

File: Dockerfile.oracle [hunk 2]
```diff
@@ -76,6 +76,8 @@ RUN find config -name '*.sample' | xargs -I{} sh -c 'cp $1 ${1%.*}' -- {}
 RUN mkdir /tmp/config
 RUN cp -r /opt/homs/config/* /tmp/config
 
+RUN yarn install && bundle exec rails assets:precompile && rm -rf /opt/homs/node_modules/
```
Comment (TimAle) [inline Dockerfile.oracle]: Not the same place as in the `Dockerfile`.

**Comment by mxsmirnoff** [comment]
Updated

**Pull request merged**